### DISCUSSION TITLE
Результат работы скрипта не соответствует информации в кадастровой выписке

### DISCUSSION THREAD

**Discussion by AndrewKirikoff** [rendrom/rosreestr2coord#76] [General]
Подскажите пожалуйста. В описание к скрипту написано, что Результат работы скрипта не соответствует информации в кадастровой выписке. Это получается что все координаты участков не соответствуют ? Получается смысла использовать этот скрипт нет если надо получить участок, могут быть проблемы с пересечением и выходом за какие-либо границы?

**Comment by rendrom** [comment]
Скрипт предоставляет приблизительные границы участков. Автоматизирует процесс обведения контуров с публичной карты росреестра. Нет никаких гарантии в работе этой библиотеки.
